### DISCUSSION TITLE
GitHub Discussions by-label and categories

### DISCUSSION THREAD

**Discussion by damian-krawczyk** [badges/shields#6956] [Ideas]
Can you add a badge for GitHub Discussion to display counts per label, like you have for issues?

![GitHub Discussions](https://img.shields.io/github/discussions/badges/shields)

![GitHub issues](https://img.shields.io/github/issues/badges/shields)

![GitHub issues by-label](https://img.shields.io/github/issues/badges/shields/service-badge)

GitHub Discussions provides categories, maybe it could be possible to create a badge for Discussions per categories?

**Comment by calebcartwright** [comment]
Haven't gone back through in detail but believe the existing feature request for discussions badges likely has some relevant context

https://github.com/badges/shields/issues/6047
